packages/iTwinUI-react/src/core/ComboBox/ComboBox.test.tsx

```diff
@@ -10,6 +10,7 @@ import { SvgCaretDownSmall } from '@itwin/itwinui-icons-react';
 import { MenuItem } from '../Menu';
 import { StatusMessage } from '../StatusMessage';
 import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
```

Comment:
Why it is imported not from testing-library?

Reply:
Ask vscode autoimport. Fixed